src/test/java/core/basesyntax/ProcessServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.ProcessingService;
+import core.basesyntax.service.impl.ProcessingServiceImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.operations.BalanceHandler;
+import core.basesyntax.strategy.operations.OperationHandler;
+import core.basesyntax.strategy.operations.PurchaseHandler;
+import core.basesyntax.strategy.operations.ReturnHandler;
+import core.basesyntax.strategy.operations.SupplyHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ProcessServiceTest {
+    private static OperationStrategy strategy;
+    private static ProcessingService processingService;
+
+    @BeforeClass
+    public static void setUp() {
+        Map<Operation, OperationHandler> operationHandlerMap = new HashMap<>();
+        operationHandlerMap.put(Operation.BALANCE, new BalanceHandler());
+        operationHandlerMap.put(Operation.PURCHASE, new PurchaseHandler());
+        operationHandlerMap.put(Operation.RETURN, new ReturnHandler());
+        operationHandlerMap.put(Operation.SUPPLY, new SupplyHandler());
+        strategy = new OperationStrategyImpl(operationHandlerMap);
+        processingService = new ProcessingServiceImpl();
+    }
+
+    @Test
+    public void processingService_removeListWithHeading_ok() {
```

Comment:
```suggestion
    public void removeHeading_threeElements_ok() {
```
The first part must be the name of the testing method